engine/src/chunk.cpp

```diff
@@ -2220,6 +2220,12 @@ bool MCChunk::set(MCExecContext &ctxt, Preposition_type p_type, MCExecValue p_va
             
             MCInterfaceExecPutIntoField(ctxt, *t_string, p_type, t_obj_chunk);
         }
+        else if (t_obj_chunk . object -> gettype() == CT_CARD)
```

Comment:
@livecodepanos why not just cut and paste from the `MCValueRef` variant of this method:
```
// AL-2014-08-04: [[ Bug 13081 ]] 'put into <chunk>' is valid for any container type
                switch (t_obj_chunk . object -> gettype())
                {
                    case CT_BUTTON:
                    case CT_IMAGE:
                    case CT_AUDIO_CLIP:
                    case CT_VIDEO_CLIP:
                        MCInterfaceExecPutIntoObject(ctxt, *t_string, p_type, t_obj_chunk);
                        break;
                    default:
                        ctxt . LegacyThrow(EE_CHUNK_SETNOTACONTAINER);
                        break;
                }
```
I don't think you need this as it's covered further down 
```
MCValueRelease(t_obj_chunk . mark . text);
return false;
```

Reply:
Thanks @montegoulding 👍